openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -216,16 +217,40 @@ def _filter_fields(self, field_whitelist, serialized_account):
         return visible_serialized_account
 
 
+class UserAccountDisableHistorySerializer(serializers.ModelSerializer):
+    """
+    Class that serializes User account disable history
+    """
+
+    disabled_by = serializers.SerializerMethodField()
+
+    class Meta(object):
+        model = UserAccountDisableHistory
+        fields = ("created", "comment", "disabled", "disabled_by")
+
+    def get_disabled_by(self, disable_history):
+        return disable_history.by.username
+
+
 class AccountUserSerializer(serializers.HyperlinkedModelSerializer, ReadOnlyFieldsSerializerMixin):
     """
     Class that serializes the portion of User model needed for account information.
     """
+    disable_history = serializers.SerializerMethodField()
+
     class Meta(object):
         model = User
-        fields = ("username", "email", "date_joined", "is_active")
-        read_only_fields = ("username", "email", "date_joined", "is_active")
+        fields = ("username", "email", "date_joined", "is_active", "disable_history")
+        read_only_fields = ("username", "email", "date_joined", "is_active", "disable_history")
         explicit_read_only_fields = ()
 
+    def get_disable_history(self, user):
+        """ Return a list of the user's disable history """
+        disable_history = UserAccountDisableHistory.objects.filter(
+            user=user
+        ).order_by("-created")
```

Comment:
We can add order_by in the models Meta class?

Reply:
How?

Reply:
```
class Meta: 
    ordering = ['-created']
```
https://docs.djangoproject.com/en/1.11/ref/models/options/#ordering

Reply:
Will this not apply to the User model?